docs/modules/ROOT/pages/leader-election.adoc

```diff
@@ -25,3 +25,76 @@ To specify the name of the configmap used for leader election use the following
 ----
 spring.cloud.kubernetes.leader.config-map-name=leader
 ----
+
+'''
+
+There is another way you can configure leader election and it comes with native support in the fabric8 library. In the long run, this will be the default way to configure leader election, while the previous one will be dropped. You can treat this one much like the JDKs "preview" features.
+
+To be able to use it, you first need to disable the old implementation:
+
+[source]
+----
+spring.cloud.kubernetes.leader.enabled=false
```

Comment:
Could we disable this if the new one is enabled so folks dont need to set 2 properties?